### DISCUSSION TITLE
[Release] 1.3.1

### DISCUSSION THREAD

**Discussion by oxedions** [bluebanquise/bluebanquise#473] [General] [answered]
We need to release an 1.3.1.

Since this release is a bug fix only from 1.3.0, does any one see a missing bug that should be backported from master to stable 1.3 ?

**Comment by btravouillon** [comment]
I opened new backport PRs: https://github.com/bluebanquise/bluebanquise/pulls?q=is%3Aopen+is%3Apr+label%3Abackport

3 PRs still need backport: https://github.com/bluebanquise/bluebanquise/pulls?q=is%3Apr+label%3Aneeds-backport

Code for #434 is ready, I will commit and push before the end of this week.

**Reply by bouriquet** [reply]
#361 already backported with #376

**Reply by btravouillon** [reply]
#434 is ready for review.

**Reply by oxedions** [reply]
Thanks a lot !!

**Reply by btravouillon** [reply]
Last PR to merge is #490 (backport of #434) then I believe we will be ready for 1.3.1.

**Reply by oxedions** [reply]
Merged. I think we are set. If no more major fix before tomorrow, lets release the 1.3.1.

**Comment by oxedions** [comment, marked as answer]
Ok, I will release the 1.3.1.

**Reply by oxedions** [reply]
Released !
I will build the rpms and upload them on github.
